aptos-move/framework/aptos-token/sources/property_map.move

```diff
@@ -46,9 +48,23 @@ module aptos_token::property_map {
     public fun empty(): PropertyMap {
         PropertyMap{
             map: simple_map::create<String, PropertyValue>(),
+            keys: vector::empty<String>(),
         }
     }
 
+    public fun copy_map(list: &PropertyMap): PropertyMap {
```

Comment:
What is the use case for this? This seems potentially expensive to do on chain if the PropertyMap is large enough.

Reply:
mainly used by token module since I don't want to add copy to SimpleMap.  You can refer to the highlighted change in token.move.

Reply:
why not just make PropertyMap copy?

Reply:
BTW, the correct name for this operation is `clone`, not copy.